src/app/_common/user/UserConsentModal.jsx

```diff
@@ -14,22 +14,28 @@ function UserConsentModal({ onClose, onTermsAccepted }) {
     const [checked, check, uncheck] = useToggle();
     const [fetchStatus, setFetchStatus] = useState(FetchStatus.NOT_FETCHED);
 
-    function createUser() {
+    async function createUser() {
         setFetchStatus(FetchStatus.IS_FETCHING);
 
-        UserAPI.post("api/user", {
-            acceptedTerms: "sok_v1",
-        })
-            .then((response) => {
-                setFetchStatus(FetchStatus.SUCCESS);
-                if (onTermsAccepted) {
-                    onTermsAccepted();
-                }
-                updateUser(response);
-            })
-            .catch(() => {
-                setFetchStatus(FetchStatus.FAILURE);
-            });
+        const res = await fetch("/stillinger/api/user", {
+            method: "POST",
+            body: JSON.stringify({ acceptedTerms: "sok_v1" }),
+        });
```

Comment:
Siden dette skjer i nettleseren så kan noen ganger fetch-kallet i seg selv feile, f.eks ved offline. Da vil du ikke få noen response og resten av koden krasjer. Kan f.eks fixes sånn:

try {
   const res = await fetch("/stillinger/api/user" ....
   if if (!res.ok) {
       ...
       ...
} catch (err) {
    setFetchStatus(FetchStatus.FAILURE);
}

Reply:
men, mulig vi ikke trenger dette om vi har en form for error boundary "lengre opp" slik at feil fanges og feilside vises.